--- nghttp2-1.3.0/python/Makefile.am.orig	2015-08-30 12:02:56.000000000 +0200
+++ nghttp2-1.3.0/python/Makefile.am	2015-09-06 20:09:36.208853530 +0200
@@ -33,7 +33,7 @@
 	$(PYTHON) setup.py build
 
 install-exec-local:
-	$(PYTHON) setup.py install --prefix=$(DESTDIR)$(prefix)
+	$(PYTHON) setup.py install --root=$(DESTDIR) --prefix=$(prefix)
 
 uninstall-local:
 	rm -f $(DESTDIR)$(libdir)/python*/site-packages/nghttp2.so
